### PR TITLE
Fix requires

### PR DESCRIPTION
**Pull request by srikiraju**
@khanduri @namanbharadwaj 
Trying to use pip-compile/sync. This comes up as a dependency breakage. How do we build this package?

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
     license='MIT',
     packages=find_packages(exclude='tests'),
     package_data={'README': ['README.md']},
-    install_requires=['requests==2.2.1'],
+    install_requires=['requests>=2.2.1'],
```
Comment (khanduri) [inline setup.py:20]: Why do we want this change? 
I was under the impression that its good practice to lock the package dependencies. Having it marked to "anything greater" may cause some future push to break that might not be fully backwards compatible. Right?

Comment (metalogical) [reply setup.py:20]: Actually, the best thing to do is to use inequalities for package dependencies, and equalities for app dependencies. See Srik/Jim/my thoughts on the topic here https://jira.team.affirm.com/browse/INFRA-2643

Comment (metalogical) [reply setup.py:20]: however, we probably also want to specify `requests<3` here

Comment (srikiraju) [reply setup.py:20]: I know...we all love the requests package 🤣

**Review by metalogical — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (srikiraju) [reply setup.py:20]: Also https://github.com/plaid/plaid-python/pull/77

**Comment by srikiraju** [comment]
Did we manually upload this package to artifactory using twine?

**Comment by metalogical** [comment]
I manually uploaded it; don't remember if I used twine or not, but that should work.

**Comment by metalogical** [comment]
can we bump the affirm version on this package?

**Comment by srikiraju** [comment]
yea, gonna bump so we don't affect current release

**Pull request merged**